### PR TITLE
Extend IUser interface by organization name for tiger BE

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: NAS-2748

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-backend-tiger/src/convertors/fromBackend/UsersConverter.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // (C) 2019-2021 GoodData Corporation
-import { IUser } from "@gooddata/sdk-backend-spi";
+import { IUser } from "@gooddata/sdk-model";
```
Comment (no23reason) [inline libs/sdk-backend-tiger/src/convertors/fromBackend/UsersConverter.ts:2]: Merge these two imports please.

File: libs/sdk-model/src/user/index.ts [hunk 2]
```diff
@@ -40,6 +40,12 @@ export interface IUser {
      * Last name - when backend implementations supports it.
      */
     lastName?: string;
+
+    /**
+     * Organization name - when backend implementations supports it.
+     */
+
```
Comment (no23reason) [inline libs/sdk-model/src/user/index.ts:47]: Please remove this empty line

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6245/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/8032/ : SUCCESS in 18m 38s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2052

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1801/ : SUCCESS in 2m 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2082/ : SUCCESS in 1m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/883/ : SUCCESS in 14m 45s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/8034/ : SUCCESS in 18m 33s

**Pull request merged**